app/components/Menu/index.js

```diff
@@ -48,12 +98,13 @@ function Menu() {
           </NavLink>
         </NavItem>
         <NavItem>
-          <NavLink to="/add-event">
+          <Button to="/add-event">
             <FormattedMessage {...messages.add} />
-          </NavLink>
+            {/* QA: kady messages wpada w span, da się temu jakoś zapobiec? */}
```

Comment:
co masz na myśli ?

Reply:
edit: ok ok czaje

Reply:
```
        <FormattedMessage {...messages.add}>
              {msg => <div>{msg}</div>}
            </FormattedMessage>
```

please read carefully [children as function](https://www.robinwieruch.de/react-pass-props-to-component)

Reply:
Right! I'll check this! Thanks!